mentorship-code-of-conduct.html

```diff
@@ -6,6 +6,56 @@
+        <h2>Code Of Conduct for Mentors</h2>
+        <ol>
+            <li><em>Women Coding Community is an inclusive community</em>, dedicated to providing an empowering experience for everyone who participates in or supports our community, regardless of gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, ethnicity, age, religion, socioeconomic status, caste, creed, political affiliation, or preferred programming language(s).</li>
+            <li>Cancel or reschedule appointments with a minimum of 24 hours prior notice by providing a reason by preferable channel (email, slack or anything else you both choose as the channel of communication) to the mentee.</li>
+            <li>No-shows are unacceptable for mentors and mentees. No-shows should be informed to the Mentorship Programme Team by <a href="https://bit.ly/women_coding_community">Slack</a> channel #mentorship. The Mentorship Programme Team will evaluate case by case and this is one of the possible actions that could result in exclusion from the current and future mentorship program cycles. </li>
+            <li>No-replies by email or slack to the Mentorship Programme Team and/or Mentee in a week are unacceptable during the program. Participants who applied but failed to reply without any reason provided will be banned from the current cycle.</li>
+            <li>Create a <a href="https://bit.ly/women_coding_community">Slack Account</a>, join #mentorship channel and check slack messages in the program to remain updated with all communication.</li>
+            <li>Update notion attendance page (template provided by the organization).</li>
+            <li>Provide information, guidance, constructive comments, and reflective listening to the mentee.</li>
+            <li>Provide a confidential and personalised source of career advice, support, and guidance to the mentee.</li>
+            <li>Whenever you see something is not going in the direction you want with the mentorship programme, provide feedback by <a href="https://bit.ly/women_coding_community">slack</a> to the Mentorship Programme Team.</li>
+            <li>Attending the Mentor's Catch Up sessions is crucial for the programme's improvement:
+                <ul>
+                    <li>It is <b>mandatory</b> to provide feedback using google forms shared with you before the session.</li>
+                    <li>If you can't attend the session, explain your reason in reply to the invitation.</li>
+                </ul>
+            </li>
+            <li>Discussions between you and your mentee <b>are considered to be confidential</b>. Be careful about sensitive personal issues. Do not share any specific content of the sessions or personal data without the permission of your mentee. </li>
+            <li>Please ensure that your profile on the mentorship page is kept up-to-date.</li>
+        </ol>
+    </div>
+    
+    <div class="col-12 section global-code">
```

Comment:
This 'global-code' class seems unnecessary. I suggest remove it, if it's not in use :)

Reply:
```suggestion
    <div class="col-12 section">
```